app/1.0/docs/devguide/data-binding.md

```diff
@@ -59,18 +91,21 @@ For example:
 syntax. For more information, see [Binding to native element attributes](#native-binding).
 { .alert .alert-info }
 
+If the property being bound is an object or an array, both elements get a reference to the **same
+object**. This means that either element can change the object and a true one-way binding is
+impossible. For more information, see [Data flow for objects and
+arrays](data-system#data-flow-objects-arrays).
 
-### Binding to text content
+### Bind to text content
 
-To bind to a child element's `textContent`, you can simply include the
-annotation inside the child element.
+To bind to a target element's `textContent`, you can simply include the
+annotation or compound binding inside the target element.
 
 ```
 <dom-module id="user-view">
 
     <template>
```

Comment:
Whoop. Not sure how the weird indentation got in there, but I think I got it all.